fuel/app/classes/controller/media.php

```diff
@@ -38,26 +43,49 @@ public function get_render($asset_id, $size='original')
 		return '';
 	}
 
+	/*
+		public function get_import()
+		{
+			// Validate Logged in
+			if (\Service_User::verify_session() !== true) throw new HttpNotFoundException;
+
+			Css::push_group(['core', 'media_import']);
+			Js::push_group(['angular', 'jquery', 'materia', 'author', 'dataTables']);
+
+			$this->inject_common_js_constants();
+
+			$theme = Theme::instance();
+			$theme->set_template('layouts/main');
+			$theme->get_template()
+				->set('title', 'Media Catalog')
+				->set('page_type', 'import');
+
+			$theme->set_partial('footer', 'partials/angular_alert');
+			$theme->set_partial('content', 'partials/catalog/media');
+
+			return Response::forge($theme->render());
+		}
+	*/
+
 	public function get_import()
 	{
 		// Validate Logged in
 		if (\Service_User::verify_session() !== true) throw new HttpNotFoundException;
 
-		Css::push_group(['core', 'media_import']);
-		Js::push_group(['angular', 'jquery', 'materia', 'author', 'dataTables']);
-
+		// Require for injecting array of uri
 		$this->inject_common_js_constants();
 
-		$theme = Theme::instance();
-		$theme->set_template('layouts/main');
-		$theme->get_template()
+		$this->theme = Theme::instance();
+		$this->theme->set_template('layouts/react');
+		$this->theme->get_template()
 			->set('title', 'Media Catalog')
 			->set('page_type', 'import');
 
-		$theme->set_partial('footer', 'partials/angular_alert');
-		$theme->set_partial('content', 'partials/catalog/media');
+		Css::push_group(['media_import']);
+		Js::push_group(['react', 'media']);
 
-		return Response::forge($theme->render());
+		// Forge is needed to render content of the iframe.
+		return Response::forge($this->theme->render());
```

Comment:
I would double-check if this is the case - I think the `forge()` method can be removed completely with the transition to react. If you look at other controller methods, `forge()` isn't referenced.